migration/src/m0000630_create_product_version_range.rs

```diff
@@ -0,0 +1,76 @@
+use sea_orm_migration::prelude::*;
+
+use crate::{m0000290_create_product::Product, UuidV4};
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(ProductVersionRange::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(ProductVersionRange::Id)
+                            .uuid()
+                            .not_null()
+                            .default(Func::cust(UuidV4))
+                            .primary_key(),
+                    )
+                    .col(
+                        ColumnDef::new(ProductVersionRange::ProductId)
+                            .uuid()
+                            .not_null(),
+                    )
+                    .col(
+                        ColumnDef::new(ProductVersionRange::VersionRangeId)
+                            .uuid()
+                            .not_null(),
+                    )
+                    .col(ColumnDef::new(ProductVersionRange::CpeKey).string())
+                    .foreign_key(
+                        ForeignKey::create()
+                            .from_col(ProductVersionRange::ProductId)
+                            .to(Product::Table, Product::Id)
+                            .on_delete(ForeignKeyAction::Cascade),
+                    )
+                    .foreign_key(
+                        ForeignKey::create()
+                            .from_col(ProductVersionRange::VersionRangeId)
+                            .to(VersionRange::Table, VersionRange::Id)
+                            .on_delete(ForeignKeyAction::Cascade),
+                    )
+                    .to_owned(),
+            )
+            .await?;
+
+        Ok(())
+    }
+
+    async fn down(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .drop_table(Table::drop().table(ProductVersionRange::Table).to_owned())
+            .await?;
+
+        Ok(())
+    }
+}
+
+#[derive(DeriveIden)]
+pub enum ProductVersionRange {
```

Comment:
nit: we remove `pub` from all Iden enums so they don't accidentally get imported into other migrations.